### PR TITLE
Add common documents

### PR DESCRIPTION
**Pull request by carolynvs**
Add common documents that can be linked to from other cnabio repositories and shared, instead of copy/pasting. This will allow us to maintain them in a single location without having drift between the repositories.

- Contributing
- Governance

This preserves the governance doc as-is. In a separate PR, I will suggest changes to match our current practices.

### CODE REVIEW

**Review by flynnduism — APPROVED** [review]

File: docs/shared/CONTRIBUTING.md
Comment (technosophos) [inline docs/shared/CONTRIBUTING.md]: "...do their best _to_ regularly make..."?

File: docs/shared/CONTRIBUTING.md
Comment (technosophos) [inline docs/shared/CONTRIBUTING.md]: nit: "before you spend"

File: docs/shared/CONTRIBUTING.md
Comment (technosophos) [inline docs/shared/CONTRIBUTING.md:20]: I like that clarification. I recently contributed a one-word spelling fix to a repo, and was super anxious that they would close the PR because I didn't first file an issue.

File: docs/shared/CONTRIBUTING.md
Comment (technosophos) [inline docs/shared/CONTRIBUTING.md]: I kinda feel like "to be nice" here might backfire. Sounds like "once your draft is done, the GLOVES COME OFF!" 🥊

**Review by technosophos — APPROVED** [review]
I left a few small nits, but nothing blocking.

File: docs/shared/CONTRIBUTING.md
Comment (carolynvs) [reply docs/shared/CONTRIBUTING.md]: bwhahahahha

**Comment by carolynvs** [comment]
I've updated the doc with @technosophos 's suggested changes.

File: docs/shared/governance.md
Comment (radu-matei) [inline docs/shared/governance.md]: I know this document is moved in its current form from the original repo - just wanted to point  out to update the project-specific references (_updating Duffle_, for example).

Not blocking this PR.

**Comment by technosophos** [comment]
Re-approved. :shipit:  Thanks!

File: docs/shared/governance.md
Comment (technosophos) [reply docs/shared/governance.md]: I saw that, but assumed each project would just change it to their thing.

File: docs/shared/CONTRIBUTING.md
Comment (chris-crone) [inline docs/shared/CONTRIBUTING.md:21]: nit: Maybe just tell people to use draft PRs. These are safer as the merge button isn't clickable.

File: docs/shared/governance.md
Comment (chris-crone) [reply docs/shared/governance.md]: Maybe something like:

```suggestion
[Project maintainers](CODEOWNERS) are responsible for activities around maintaining and updating <PROJECT NAME>, <PROJECT DESCRIPTION>. Final decisions on the project reside with the project maintainers.
```

**Review by chris-crone — APPROVED** [review]
LGTM, thanks @carolynvs !

File: docs/shared/CONTRIBUTING.md
Comment (carolynvs) [reply docs/shared/CONTRIBUTING.md:21]: I've been telling people about both because they often open a regular PR by accident, or they think they have it ready then change their mind, and aren't sure how to say "oops I need more time".

**Comment by carolynvs** [comment]
I've removed the references to Duffle and made them generic. I want to avoid making any substantive changes in this PR (as per the OP) because the intent is to copy over the existing docs as-is. Then in a series of follow-ups we can alter them to fit how we want to govern the projects.

**Review by chris-crone — APPROVED** [review]
LGTM

**Pull request merged**
